src/main/java/com/mongodb/kafka/connect/util/jmx/SourceTaskStatisticsMBean.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2022-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.kafka.connect.util.jmx;
+
+/**
+ * <p>A standard MBean interface for a {@link com.mongodb.kafka.connect.source.MongoSourceTask}.</p>
+ *
+ * <p>This interface is NOT part of the public API.  Be prepared for non-binary compatible changes in minor releases.</p>
+ *
+ * @since TODO
+ */
+public interface SourceTaskStatisticsMBean {
```

Comment:
We should distinguish in some way between the two phases of the source task: the copy phase and the change stream phase.  These stats will be misleading if you don't know what phase you're in.  As one example, getMore latencies of 5 seconds are normal in the change stream phase, but highly unusual in the copy phase.  If you can't tell the difference, it's going to be confusing to customers and support engineers.

Reply:
Agreed. It seems there are also some stats (total number of records sent) the user might prefer not to split. We could:

1. Create a separate property for each stat.
2. When the task switches from "copy" to "change stream", switch to a second MBean.
3. Switch the MBean as in #2, but also offer a third MBean where both phases are summed together for convenience.
4. Keep it as is, but introduce a new stat (like "records left to copy") that communicates when the copy phase is over, and assume that the user will notice this stat.

We were leaning towards #3.